www/less/widenImages.less

```diff
@@ -29,6 +20,15 @@
         max-width:50% !important;
         height:auto !important;
     }
+	
+    /* Tablet override */
+    @media (min-device-width: 768px) {
+        .wideImageOverride {
+            width:100% !important;
```

Comment:
I think the css isn't quite right now - if you drop these values to 50% and run `make grunt` the image width isn't taking on landscape iPad...